src/janus-api/__mocks__/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+* Copyright (c) [2019] SUSE Linux
```

Comment:
NP: probably Prettier will "complain" here because a missing space after the `*`. The same in the other mock.

Reply:
Another note: the rest of files has a `2015-2019` in the copyright header.

Reply:
Actually, this is a new file, so it is ok as it is IMHO. Different thing are those files under `janus-api`, although I know that we have modified other new files aswell.